packages/twenty-front/src/modules/auth/sign-in-up/components/SignInUpForm.tsx

```diff
@@ -54,6 +54,7 @@ export const SignInUpForm = () => {
   const { form } = useSignInUpForm();
 
   const {
+    userExists,
```

Comment:
I feel like the best instead of introducing userExists could have been to fix the initial design misconception

SignInUpStep = init/email/password OK
But SignInUpMode = sign-in/sign-up/invite is WRONG

You should probably limit SignInUpMode to sign-in/sign-up (which is equivalent to your userExists)
And then isInviteMode should be a boolean. That's roughly the same as what you did but you fix the structural issue in doing that :)

Reply:
Fixed it, I think its much simpler and better now :)